.github/workflows/continuous-delivery.yml

```
@@ -1,38 +1,63 @@
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+    paths:
+      - "app/**/*"
+      - "server/**/*"
+      - "config/*"
+      - "modules/*"
+      - nuxt.config.ts # File config Nuxt
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)